tests/python_functional/functional_tests/config_change/test_manipulating_config_between_reload.py

```diff
@@ -32,7 +32,7 @@ def test_manipulating_config_between_reload(config, syslog_ng):
     syslog_ng.start(config)
 
     # update positional value of file source
-    file_source.set_path("updated_input.log")
+    file_source.options['file_name'] = "updated_input.log"
```

Comment:
Thanks, I will fix this to use: `file_source.get_connection()`

Reply:
You are absolutely right. After a little testing I found a missing functional testcase (where we replace our driver resource before a reload, and after the reload we wanted to use the new resource). With this testcase the current implementation will fail. 
So we will need some method to rewrite our positional parameters in tests.
Because of this I will reinterpret my changes.